src/main/java/christmas/ExceptionList.java

```diff
@@ -0,0 +1,18 @@
+package christmas;
+
+public enum ExceptionList {
+    DATEEXCPTION("유효하지 않은 날짜입니다. 다시 입력해 주세요."),
```

Comment:
상수의 이름을 네이밍 할 때는 `_`로 단어들을 구분해 주는 것이 가독성 측면에서 훨씬 좋습니다!
그리고, 이미 해당 클래스의 이름이 `ExceptionList`이기에 `DATE_EXCEPTION`을 가져와 사용한다면 다음과 같은 코드가 될 것 같은데요.
`ExceptionList.DATE_EXCEPTION....` 이 경우 클래스 이름에서 이미 `Exception`에 관련된 정보라는 것을 확인할 수 있기에 `EXCEPTION`이라는 부분을 제거해도 문제가 없을 것 같습니다.

변수 이름이 `DATE`로만 끝나는 것이 어색하다면 `INVALID_DATE`와 같은 선택도 있을 것 같네요
```suggestion
    DATE("유효하지 않은 날짜입니다. 다시 입력해 주세요."),
```